.changeset/factory-think-command.md

```
@@ -0,0 +1,5 @@
+---
+'mastra': patch
+---
+
+Fixed Factory chat `/think` so it can set, inspect, and clear session reasoning overrides with the same command semantics as the Mastra Code TUI.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)